### PR TITLE
Route

### PR DESCRIPTION
**Pull request by muneebsaddal**

### CODE REVIEW

File: src/components/header/Header.js [hunk 1]
```diff
@@ -1,19 +1,33 @@
...
 			<HeaderButtonContainer>
 				<AddButton onClick={props.handleOpen} title="add">
 					<IoMdAdd size={26} />
 				</AddButton>
-				<FilterButton>
-					<BiFilterAlt size={25} />
-				</FilterButton>
+				<SearchButton title="search">
+					<Container>
+						{props.showSearchField && (
```
Comment (NabilShahid) [inline src/components/header/Header.js:16]: Lets create a separate component for search field.

File: src/components/addHabitDialogue/AddHabitDialogue.js [hunk 2]
```diff
@@ -27,22 +28,33 @@ function AddHabitDialogue(props) {
...
-						e.g. How many miles did you run today? How many pages
-						did you read?
-					</Content>
+				<Option title="measurable">
+					<Link
+						to="/formMeasurable"
+						style={{ textDecoration: "none", color: "black" }}
+					>
+						<Heading name="measurable">Measurable</Heading>
+						<br />
```
Comment (NabilShahid) [inline src/components/addHabitDialogue/AddHabitDialogue.js:50]: Avoid using br tags and utllize the concept of block elements in html/css

Comment (muneebsaddal) [reply src/components/addHabitDialogue/AddHabitDialogue.js:50]: fixed

File: src/components/header/Header.js (hunk 1 above)
Comment (muneebsaddal) [reply src/components/header/Header.js:16]: made a seperate component

**Pull request merged**